### ISSUE TITLE
add-on breaks NVDA initialization

### ISSUE THREAD

**Issue by codeofdusk** [NVDARemote/NVDARemote#225]
After nvaccess/nvda#11650 was merged, this add-on now prevents NVDA from starting. No apparent errors occurred in the log when NVDA log level was set to debug. Cc @ctoth, @michaelDCurran, @tspivey.

**Comment by michaelDCurran** [comment]
As it is our policy not to break add-ons except on a point 1 release, along with an addon compatibility version change,  this is NVDA's problem and we will either revert that pr or find a fix. I think this issue should be closed.

**Comment by codeofdusk** [comment]
Makes sense.
